server/routes/comments.js

```diff
@@ -0,0 +1,35 @@
+var express = require('express');
+var router = express.Router();
+var models = require('../models');
+
+/* GET all snippet comments*/
+router.get('/:snippetId/comments', function (req, res) {
+  models.Comment.findAll({
+    where: {
+      SnippetId: req.params.snippetId
+    }
+  }).then(function (comments) {
+    var mappedComments = comments.map(function (c) {
+      return c.toJson();
+    });
+    res.send(mappedComments);
+  });
+});
+
+/* POST new snippet comment*/
+router.post('/:snippetId/comments', function (req, res) {
+  var body = req.body;
+  var attributes = {
+    content: body.content,
+    SnippetId: req.params.snippetId
+  };
+  models.sequelize.transaction(function (t) {
```

Comment:
@vrael560 I don't think transaction is necessary here. I mean it works perfectly ok but it is single query and it will fullfil or faild and with or without transaction result will be the same.